source/ed/ed_reader.cpp

```diff
@@ -358,6 +359,38 @@ void EdReader::fill_contributors(nt::Data& data, pqxx::work& work){
     }
 }
 
+void EdReader::fill_frames(nt::Data& data, pqxx::work& work){
+    std::string request = "SELECT id, uri, description, system, start_date, end_date, "
+                          "contributor_id FROM navitia.frame";
+    size_t nb_unknown_contributor(0);
+    pqxx::result result = work.exec(request);
+    for(auto const_it = result.begin(); const_it != result.end(); ++const_it){
+        auto contributor_it = this->contributor_map.find(const_it["contributor_id"].as<idx_t>());
+        if(contributor_it == this->contributor_map.end()) {
```

Comment:
that should never happens no ?
I'm more favorable to throwing a stoping exception for incoherent code (because this should be handled in FusioParser)

@pbougue @kinnou02 what do you think ?